src/patternfly/components/MenuToggle/menu-toggle-toggle-icon.hbs

```diff
@@ -5,6 +5,6 @@
   {{#if @partial-block}}
     {{> @partial-block}}
   {{else}}
-    <i class="fas fa-caret-down" aria-hidden="true"></i>
+    <i class="fas fa-caret-down fa-fw" aria-hidden="true"></i>
```

Comment:
Mimics the react-icons svg size a little better. Technically this one is a little wider than the react-icons svg.